README.md

```diff
@@ -83,21 +83,21 @@ See [`:help nvim-from-vim`](https://neovim.io/doc/user/nvim.html#nvim-from-vim)
 Project layout
 --------------
 
-    ├─ ci/              build automation
-    ├─ cmake/           build scripts
-    ├─ runtime/         user plugins/docs
-    ├─ src/             application source code (see src/nvim/README.md)
-    │  ├─ api/          API subsystem
-    │  ├─ eval/         VimL subsystem
-    │  ├─ event/        event-loop subsystem
-    │  ├─ generators/   code generation (pre-compilation)
-    │  ├─ lib/          generic data structures
-    │  ├─ lua/          Lua subsystem
-    │  ├─ msgpack_rpc/  RPC subsystem
-    │  ├─ os/           low-level platform code
-    │  └─ tui/          built-in UI
-    ├─ third-party/     CMake subproject to build dependencies
-    └─ test/            tests (see test/README.md)
+    ├── ci/                   build automation
+    ├── cmake/                build scripts
+    ├── runtime/              user plugins/docs
+    ├── src/                  application source code (see src/nvim/README.md)
+    │   └── nvim/             API subsystem
+    │       ├── api/          VimL subsystem
```

Comment:
Now the labels are wrong.

I made an adjustment in https://github.com/neovim/neovim/pull/9288 .